outsource/meta.py

```diff
@@ -0,0 +1,47 @@
+# These data_types need per-chunk storage, so don't upload to rucio here!
+PER_CHUNK_DATA_TYPES = [
```

Comment:
What about records?

Reply:
`records` are not saved. So I disallow `records` being in `include_data_types`.